src/external/samba.m4

```diff
@@ -160,4 +160,19 @@ AC_CHECK_MEMBERS([struct PAC_LOGON_INFO.resource_groups], , ,
                  [[ #include <ndr.h>
                     #include <gen_ndr/krb5pac.h>
                     #include <gen_ndr/krb5pac.h>]])
+AC_CHECK_MEMBERS([struct PAC_REQUESTER_SID.sid], ,
+                 [AC_MSG_NOTICE([struct PAC_REQUESTER_SID is not available, PAC checks will be limited])],
+                 [[ #include <ndr.h>
+                    #include <gen_ndr/krb5pac.h>
+                    #include <gen_ndr/krb5pac.h>]])
+AC_CHECK_MEMBERS([struct PAC_ATTRIBUTES_INFO.flags], ,
+                 [AC_MSG_NOTICE([struct PAC_ATTRIBUTES_INFO is not available, PAC checks will be limited])],
+                 [[ #include <ndr.h>
+                    #include <gen_ndr/krb5pac.h>
+                    #include <gen_ndr/krb5pac.h>]])
+AC_CHECK_MEMBERS([struct PAC_UPN_DNS_INFO.ex], ,
```

Comment:
What sense does this check make?
It looks `PAC_UPN_DNS_INFO` is used unconditionally later on.